src/ObjectUtils/ElfFile.cpp

```diff
@@ -340,6 +345,7 @@ ErrorMessageOr<SymbolInfo> ElfFileImpl<ElfT>::CreateSymbolInfo(
 
   llvm::Expected<uint64_t> maybe_value = symbol_ref.getValue();
   if (!maybe_value) {
+    llvm::consumeError(maybe_value.takeError());
```

Comment:
do you want to make it part of the warning below? same for other places consuming the error

Reply:
Yes. Good point. Done.